### DISCUSSION TITLE
My CQZ

### DISCUSSION THREAD

**Discussion by frrobert2** [foldynl/QLog#356] [Q&A]
I am located in EM61 grid square, but for some reason when I I log a contact QLog it inserts 5 for My CQZ instead of 4.  I have not checked if the error exists for the other side of the QSO.

Thanks,

Robert

**Comment by foldynl** [comment]
Please, could you send be your callsign?

**Comment by frrobert2** [comment]
W9RWB

**Reply by foldynl** [reply]
QLog derives CQZ, ITU from callsign - the source is CTY.dat file. 

Your callsign prefix is set to CQZ 5 there. it will be necessary to amend QLog so that the operator can enter his own CQZ/ITU

**Comment by frrobert2** [comment]
I believe it is an error in the CTY.dat My call sign has a 9 which means it originated in Wisconsin, Illinois, or Indiana.  All three states are in CQZ 4.  The W can be located anywhere in the US.  So if you go by call sign my CQZ would be a 4.

Noticed I used the word originated, if I moved somewhere else in the US that was in a different CQZ, I would not have to change my call sign, so even if I was in let say moved to California, I would still be W9RWB but now my CQZ would be 3.

I think the better logic might be to use Grid Square value to determine CQZ/ITU.

**Reply by foldynl** [reply]
I would really like to use Grid to derive CQ/ITUZ, but it is not possible, because the borders of the zones are not aligned to the grids

All that remains is to fix it by adding CQZ and ITUZ to the profile.
